### ISSUE TITLE
GBM is nice ... but not really usable.

### ISSUE THREAD

**Issue by shakeyourbunny** [MikeMaximus/gbm#184]
Hi,

your (neatly) software is nice, but currently, it is not really useful.

## Bugs
- The standard Documents folder is braindead for the default backup directoy path. Use an extra distinct path.

- The first time wizard lets you select the backup path which does not carry over
- It's not really obvious for what the process and tag manager are useful for.
- No support for Windows Registry entries (there many games which store game backup information there)

## Improvements
- A "Backup all (known) games" would be useful 
- backup checked (+ select all/none) savegames button
- Indication icons and filter for games are detected (installed and/or savegames from them); the current filter system is overly complicated and has something to do with the tags... what they are for?
- Online Submission of savegame definitions would be a boon (need to verify them though)
- import of savegame definitions from other gamesave backup manager would be a boon.

**Comment by MikeMaximus** [comment]
Thanks for taking the time to let me know how you feel about GBM.

> The standard Documents folder is braindead for the default backup directoy path. Use an extra distinct > path.

The documents folder as a default is meant as a starting place to browse and a safe default for anyone that just clicks through the startup wizard without reading.  But you're right there could be a much better default folder, I should address that next release.

> The first time wizard lets you select the backup path which does not carry over

Not sure what you mean here.  I just tested this and the backup path set in the startup wizard was saved.  There is a technical issue that doesn't allow you to go back in the wizard once it's set,  but it can be easily changed in the Settings menu after the Start-up Wizard is complete.

> It's not really obvious for what the process and tag manager are useful for.

Yes the interface and some functions are not very intuitive, that has not been a design priority.  I try to keep the [manual](http://mikemaximus.github.io/gbm-web/manual.html) up to date to describe what each area of the app is for.

> No support for Windows Registry entries (there many games which store game backup information there)

[Registry support](http://mikemaximus.github.io/gbm-web/manual.html#registry) was added just a few days ago in v1.1.8.

> A "Backup all (known) games" would be useful / backup checked (+ select all/none) savegames button

This is a common request that I keep forgetting about.  It can be done on the Game Manager, but not easily.  This is something else I'll consider for the next version.

> Indication icons and filter for games are detected (installed and/or savegames from them); the current > filter system is overly complicated and has something to do with the tags... what they are for?

The [import section](http://mikemaximus.github.io/gbm-web/manual.html#import) of the manual may help.  Tags are mainly used as an optional organization feature and to help maintainers organize the official list.

> Online Submission of savegame definitions would be a boon (need to verify them though)

This is another common request and it would be useful to allow anyone to do this easily.  I haven't done this mainly because I don't want to deal with the verification of a lot of small submissions.  I do allow others to contribute to the official configuration lists via pull requests on the [GBM website](https://github.com/MikeMaximus/gbm-web) repository.

> import of savegame definitions from other gamesave backup manager would be a boon.

I agree,  this is something I should look into.
